src/main/java/com/atlassian/db/replica/api/AuroraMultiReplicaConsistency.java

```diff
@@ -61,12 +69,66 @@ public boolean isConsistent(Supplier<Connection> replicaSupplier) {
             });
     }
 
+    private boolean isConsistent_new(Supplier<Connection> replicaSupplier) {
+        return cluster.getReplicas(replicaSupplier)
+            .stream()
```

Comment:
have we tested that all consistencies for multiple replicas will be run in parallel?

Reply:
The checks will run sequentially.

Reply:
Am I right that we should raise a new issue to reduce latency ? 
Here or in internal?

Reply:
If the latency is a problem here, and you believe we can improve it by getting LSN in parallel, you can raise a new issue.

Reply:
https://github.com/atlassian-labs/db-replica/issues/144